README.md

```diff
@@ -1 +1,2 @@
-# Git-Practice
\ No newline at end of file
+# Git-Practice
+# "Adding a Change Here :)"!
```

Comment:
I don't quite like this statement. Could we try something more exciting instead:
```suggestion
# "Adding a Change Here :)"!!!!
```